### DISCUSSION TITLE
GliDOS open sourced

### DISCUSSION THREAD

**Discussion by stuaxo** [dosemu2/dosemu2#2097] [General]
About a month late, I noticed GliDOS has been open sourced
https://www.vogons.org/viewtopic.php?f=20&p=1191428#p1191428

https://www.glidos.net/code.html?lang=en

I can't see any license in the repo, which is probably just an oversight.

Maybe not useful for adding Glide here, but interesting anyway.

**Comment by stsp** [comment]
What is it?
No single word of a description on
a homepage, nor in the repo.
Why do we care?

**Reply by Torinde** [reply]
It seems to be a guest-side wrapper and host infrastructure that executes Glide commands on the host (on D3D or OpenGL as target, I assume)

**Reply by stuaxo** [reply]
The interesting thing that it seems to do, is allow replacement of textures with higher resolution ones - if dosemu2 is going to implement it's own Glide support, then it might be worth supporting the same format.

Apologies, for taking a while to come back - I wanted to check it out, but got about as far as realising it seems to need Windows 9x or NT + that I don't have a suitable VM to test it on.

**Reply by stuaxo** [reply]
When I first saw it, I assumed it was for DOS, so would have been more relevant, if it's license compatible it still might be useful to use as a reference for implementing Glide here, OTOH there seem to be a few implementations of that so maybe not as useful as I initially thought.

**Reply by stuaxo** [reply]
I do want to try this, a nice thing it seems to do, is have configuration scripts - so assuming you have a supported game, then it will install it and configure the higher resolution textures for you - having fiddled with texture replacement in other emulatiors this is a good thing.
